### DISCUSSION TITLE
02230286

### DISCUSSION THREAD

**Discussion by Khemraj9815** [kevinlin1/just-the-class#42] [Q&A]
https://www.figma.com/file/rf8yWafl7Qj9hDmQPjOx4i/Untitled?type=design&node-id=0%3A1&mode=design&t=ewCksQ3pjcGFuwPo-1

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
